addon/components/array-container.js

```diff
@@ -2,61 +2,57 @@ import 'ember-frost-bunsen/typedefs'
 
 import _ from 'lodash'
 import Ember from 'ember'
+const {A, Component} = Ember
 import computed, {readOnly} from 'ember-computed-decorators'
-
-import layout from './template'
 import PropTypeMixin, {PropTypes} from 'ember-prop-types'
-import {getLabel} from '../utils'
+import {deemberify, getLabel} from '../utils'
```

Comment:
The reason it still exists here is because if we let the store drive the state of the array components every time you type a character in an input belonging to an array item, focus is lost as Ember recreates the array components. The only way we found around this was to have the array container maintain it's own list of array items and update that based on what the store is providing. It may still be possible to rid this code of `deemberify` but I haven't verify that.